cmd/vicadmin/vicadm_test.go

```diff
@@ -85,9 +86,9 @@ func testLogTar(t *testing.T, plainHTTP bool) {
 
 	var res *http.Response
 	if !plainHTTP {
-		res, err = insecureClient.Get(fmt.Sprintf("https://localhost:%d/container-logs.tar.gz", port))
+		res, err = insecureClient.Get(fmt.Sprintf("https://root:thisisinsecure@localhost:%d/container-logs.tar.gz", port))
 	} else {
-		res, err = http.Get(fmt.Sprintf("http://localhost:%d/container-logs.tar.gz", port))
+		res, err = http.Get(fmt.Sprintf("http://root:thisisinsecure@localhost:%d/container-logs.tar.gz", port))
 	}
```

Comment:
Should also have a case that tests invalid credentials returns http.StatusUnauthorized